sdk/core/Azure.Core.TestFramework/src/RecordMatcher.cs

```diff
@@ -58,6 +60,16 @@ public RecordMatcher(bool compareBodies = true)
             "x-ms-correlation-request-id"
         };
 
+        /// <summary>
+        /// Query parameters whose values can change between recording and playback without causing URI matching
+        /// to fail. The presence or absence of the query parameter itself is still respected in matching.
+        /// </summary>
+        public HashSet<string> VolatileQueryParameters = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
+        {
+        };
+
+        private const string VolatileValue = "Volatile";
```

Comment:
Should we use `Sanitized` here since we're effectively removing it?  I don't think anyone will ever see this value so it's probably fine either way.

Reply:
Eh, I figured using Sanitized would get some pushback as we are not actually sanitizing sensitive data, which is how this term is used throughout the test framework.